packages/import-cost/test/index.spec.js

```diff
@@ -85,7 +85,8 @@ describe('importCost', () => {
   it('calculates size of scoped import in typescript', () => test('import-scoped.ts', '@angular/core'));
   it('calculates size of shaken import in javascript', () => test('import-shaken.js', 'react', 500, 1000));
   it('calculates size of shaken import in typescript', () => test('import-shaken.ts', 'react', 500, 1000));
-  it('calculates size without exports', () => test('import-exports.js', 'wix-style', 500, 1000));
+  xit('calculates size without exports', () => test('import-exports.js', 'wix-style', 500, 1000));
```

Comment:
Disabled this test because it does not appear to be working; the failure was masked by the hard-coding of React in `externals`

Reply:
this test is supposed to verify that react is hard coded in externals :)

Reply:
In that case, the new test should cover this functionality.

Reply:
I decided to leave the hardcoded default deps there as well because there are many cases where lib depends on react/lodash not through peer dep and app puts react/lodash in externals.